docs/pages/application-access/controls.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Application Access Access Controls
+description: Role-based access control (RBAC) for Teleport Application Access
+---
+
+# Application Access Role-Based Access Control
+
+## Assigning Labels to Applications
+
+Teleport Application Access uses labels to allow/deny access to the proxied
+web applications.
+
+An application can be assigned static and dynamic labels via configuration:
+
+```yaml
+apps:
+- name: "grafana"
+  uri: "http://localhost:3000"
+  # Static labels.
+  labels:
+    env: "prod"
+    group: "metrics"
+  # Dynamic labels.
+  commands:
+  - name: "arch"
+    command: ["uname", "-p"]
+    period: 1m0s
+```
+
+## Configuring Application Labels in Roles
+
+User roles can be configured to allow or deny access to applications with
+specific labels via `app_labels` property.
+
+For example, this role will grant access to all applications from the group
+"metrics", except for the production ones:
+
+```yaml
+kind: role
+version: v3
+metadata:
+  name: dev
+spec:
+  allow:
+    # At least one login must be present (even if invalid), otherwise user
+    # won't be allowed to log into the web UI.
+    logins: ["keep-this-value-here"]
+    app_labels:
+      group: "metrics"
+  deny:
+    app_labels:
+      env: "prod"
+```
+
+## Integrating with Identity Providers
+
+Role application labels can be populated dynamically based on the claims and
```

Comment:
Nit: If you switch this sentence to active voice, and split this sentence into two parts, it would make it easier for me to follow.